nautobot_chatops/dispatchers/base.py

```diff
@@ -37,6 +38,38 @@ def __init__(self, context=None):
         """Init this Dispatcher with the provided dict of contextual information (which will vary by app)."""
         self.context = context or {}
 
+    def _get_cache_key(self) -> str:
+        """Key generator for the cache, adding the plugin prefix name."""
+        # Using __file__ as a key customization within the cache
+        key_string = "-".join([__file__, self.context.get("user_name")])
+        return hashlib.md5(key_string.encode("utf-8")).hexdigest()  # nosec
```

Comment:
Is this intended as a security measure of sorts? Or else, why not just use the key_string directly as the key?

Reply:
it is not needed, I used this formula to keep a consistent format of the keys, but it's not needed in this case